tests/contrib/aiobotocore/py35/test.py

```diff
@@ -36,22 +37,35 @@ def tearDown(self):
                 await stream.read()
 
         traces = self.tracer.writer.pop_traces()
-        eq_(len(traces), 2)
-        eq_(len(traces[0]), 1)
-        eq_(len(traces[1]), 1)
-
-        span = traces[0][0]
-        eq_(span.get_tag('aws.operation'), 'GetObject')
-        eq_(span.get_tag('http.status_code'), '200')
-        eq_(span.service, 'aws.s3')
-        eq_(span.resource, 's3.getobject')
-
-        read_span = traces[1][0]
-        eq_(read_span.get_tag('aws.operation'), 'GetObject')
-        eq_(read_span.get_tag('http.status_code'), '200')
-        eq_(read_span.service, 'aws.s3')
-        eq_(read_span.resource, 's3.getobject')
-        eq_(read_span.name, 's3.command.read')
-        # enforce parenting
-        eq_(read_span.parent_id, span.span_id)
-        eq_(read_span.trace_id, span.trace_id)
+
+        version = aiobotocore.__version__.split(".")
+        pre_08 = int(version[0]) == 0 and int(version[1]) < 8
+        if pre_08:
+            eq_(len(traces), 2)
+            eq_(len(traces[0]), 1)
+            eq_(len(traces[1]), 1)
+
+            span = traces[0][0]
+            eq_(span.get_tag('aws.operation'), 'GetObject')
+            eq_(span.get_tag('http.status_code'), '200')
+            eq_(span.service, 'aws.s3')
+            eq_(span.resource, 's3.getobject')
+
+            read_span = traces[1][0]
+            eq_(read_span.get_tag('aws.operation'), 'GetObject')
+            eq_(read_span.get_tag('http.status_code'), '200')
+            eq_(read_span.service, 'aws.s3')
+            eq_(read_span.resource, 's3.getobject')
+            eq_(read_span.name, 's3.command.read')
+            # enforce parenting
+            eq_(read_span.parent_id, span.span_id)
+            eq_(read_span.trace_id, span.trace_id)
+        else:
```

Comment:
Can you add a brief dev note here for the change in number of spans?

Reply:
Done.